atom/src/atomlist.cpp

```diff
@@ -71,8 +93,14 @@ init_methods()
         py_bad_internal_call( "failed to load list 'reverse' method" );
         return false;
     }
+#if PY_VERSION_HEX >= 0x03070000
+    // typedef _PyCFunctionFast func_t;
```

Comment:
Can just the typedef be redefined here here?

Using `typedef _PyCFunctionFastWithKeywords func_t;` builds and passes tests.

Reply:
Actually it might be cleaner to define the types the top and use reinterpret cast where needed.
```
#if PY_VERSION_HEX >= 0x03070000
    typedef _PyCFunctionFast func_t;
    typedef _PyCFunctionFastWithKeywords kwfunc_t;
#else
    typedef PyCFunction func_t;
    typedef PyCFunctionWithKeywords kwfunc_t;
#endif
```

Reply:
That does seem nicer. I will have a look.